src/SprykerMiddleware/Zed/Process/Communication/Plugin/AbstractTranslatorStagePlugin.php

```diff
@@ -16,6 +17,6 @@
     public function process($payload, LoggerInterface $logger)
     {
         return $this->getFacade()
-            ->translate($payload, $this->getDictionary(), $logger);
+            ->translate($payload, $this->getTranslatorConfig(), $logger);
```

Comment:
Where from does this function come?
Looks like it should be an abstract function here.